src/popups/HandicapPopup/HandicapPopupElement.js

```diff
@@ -103,18 +104,22 @@ const renderLinks = (intialText) => {
   ]);
   const emailMatches = searchMatches(intialText, [emailTester]);
   const urlMatches = searchMatches(intialText, [urlTester]);
+  let matched = false;
 
   let replaced = [intialText];
   if (telMatches.length) {
+    matched = true;
     replaced = replaceLinks(replaced, telMatches, telTo);
   }
   if (emailMatches.length) {
+    matched = true;
     replaced = replaceLinks(replaced, emailMatches, mailTo);
   }
   if (urlMatches.length) {
+    matched = true;
     replaced = replaceLinks(replaced, urlMatches, urlHref);
   }
-  return replaced;
+  return matched ? replaced : <span tabIndex={0}>{[intialText]}</span>;
 };
```

Comment:
This method looks weird. There is a lot of redundant code. Could you find an alternative?

Reply:
This method is not related to the current ticket. If you have a improvment idea I would be glad else I can't spend time on this feature sorry